### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by sijad**
first argument passed to `tiny-warning` should be falsy to warn.

https://github.com/alexreardon/tiny-warning/blob/dc65095d4e27290869781c778950ed82a91de866/src/index.js#L9

### CODE REVIEW

**Comment by jaredpalmer** [comment]
Test failed because of my other type errors. I just upgraded TS and @types/react in the hooks branch

**Comment by jaredpalmer** [comment]
Merging with latest hooks

**Comment by jaredpalmer** [comment]
Thanks for this! Going to merge and cut a release

File: src/Formik.tsx [hunk 1]
```diff
@@ -314,10 +314,8 @@ export function useFormik<Values = object>({
         activeElement instanceof HTMLButtonElement
       ) {
         warning(
-          !!(
```
Comment (buzinas) [inline src/Formik.tsx]: If it was `!!`, shouldn't it be `!` now? It seems that this will start triggering false positives.

Comment (sijad) [reply src/Formik.tsx]: nice catch, thanks

**Comment by jaredpalmer** [comment]
I went down a rabbit hole on this one. We need to basically rearchitect our tests to use `act()`, but really need Sunil's `await act()` PR to land in the next React release. Until then, our tests will have warnings

File: src/Formik.tsx (hunk 1 above)
Comment (jaredpalmer) [reply src/Formik.tsx]: The test fails if you make this change

**Pull request merged**